### PR TITLE
libvirt 2.0.0

### PR DESCRIPTION
**Pull request by nijikon**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/libvirt.rb [hunk 1]
```diff
@@ -28,6 +28,11 @@ class Libvirt < Formula
     cause "Undefined symbols when linking"
   end
 
+  patch :p0 do
```
Comment (DomT4) [inline Formula/libvirt.rb]: Has this been reported/submitted upstream?

**Comment by GuillaumeDIDIER** [comment]
How is it going ?

**Comment by justinclift** [comment]
Looking at the Libvirt Developers mailing list archive since the 2.0.0 release, there doesn't seem to be a mention for this patch:

&nbsp; &nbsp; https://www.redhat.com/archives/libvir-list/2016-July/thread.html

Also, there's no sign of it having been applied to the upstream source:

&nbsp; &nbsp; https://libvirt.org/git/?p=libvirt.git;a=blob;f=src/util/virsystemd.h;h=93b0aae7e15fa71653a1d687dc686f6f1efa0d15;hb=HEAD

I'll report the problem on the mailing list, and see if the patch is acceptable as-is.  Kind of doubting it though, as the contents of the patch are after the:

```
#endif /* __VIR_SYSTEMD_H__ */
```

... so outside the main multiple inclusion guard.  Hopefully it'll just need to be moved inside that to be acceptable.  Will find out.

**Comment by justinclift** [comment]
As a side note, even with this patch it's barfing for me on OSX 10.9.5 with an error about curl not being found.  Will investigate.

**Comment by justinclift** [comment]
The start of the email thread, for anyone interested:

&nbsp; &nbsp; https://www.redhat.com/archives/libvir-list/2016-July/msg00613.html

**Comment by justinclift** [comment]
The commit here should work instead (replacing the one in this PR):

&nbsp; &nbsp; https://github.com/justinclift/homebrew-core/commit/e2ec6543c632b6f4803f89716c2604a8bef4b465

It includes a curl dependency (now needed), and the upstream version of the patch to fix the compile failure.

Workable? :smile:

**Comment by justinclift** [comment]
Belay that... adding a simple `test do` clause now as well.

**Comment by justinclift** [comment]
k, this one's good: https://github.com/justinclift/homebrew-core/commit/6b0681f7dc359d2e7c3248bbf92616925a73c9dd

:smile:

**Comment by justinclift** [comment]
@DomT4 Nudge?

**Comment by DomT4** [comment]
@justinclift Would you mind creating your own PR for this? It's easier to review & means we might get something mergeable sooner rather than later.

**Comment by justinclift** [comment]
Sure @DomT4.  It's here: #3301

**Comment by DomT4** [comment]
Closing in favour of https://github.com/Homebrew/homebrew-core/pull/3301.

**Pull request closed without merging**